### PR TITLE
Add sota distro variant

### PR DESCRIPTION
**Pull request by vishalbhoj**
Signed-off-by: vishal <vishal.bhoj@linaro.org>

### CODE REVIEW

**Comment by koenkooi** [comment]
Looks good to me. Can be merged after the manifest changes.

File: conf/distro/rpb-sota.conf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require conf/distro/include/rpb.inc
```
Comment (ndechesne) [inline conf/distro/rpb-sota.conf]: why don't we include:
require conf/distro/sota.conf.inc

like it seems to be recommended in meta-updater? their .inc file is doing a few other things that we aren't doing here, for example adding usrmerge

Comment (vishalbhoj) [reply conf/distro/rpb-sota.conf]: Thanks for pointing it out. It should be included. I have updated the patch.

File: conf/distro/rpb-sota.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+require conf/distro/sota.conf.inc
+
+DISTRO_NAME = "Reference-Platform-Build-SOTA"
+
+INHERIT += "sota"
+DISTRO_FEATURES_append = " sota"
```
Comment (ndechesne) [inline conf/distro/rpb-sota.conf]: if you include sota.conf.inc, then you don't need to duplicate the config manually... that's the whole point.

**Pull request merged**